TWLight/applications/views.py

```diff
@@ -651,6 +674,7 @@ def get_queryset(self, **kwargs):
         if self.request.user.is_superuser:
             base_qs = (
                 Application.objects.filter(
+                    ~Q(partner__authorization_method=Partner.BUNDLE),
```

Comment:
Is there a particular reason to use `~Q` instead of `.exclude()`?

Reply:
I did try to implement exclusion of Bundle partners with `exclude()`, but I couldn't get [this filter](https://github.com/WikipediaLibrary/TWLight/pull/401/files/22bd4f551fbacd0d8722cef8c6d2b0804f474bec#diff-dba635dfa3cab82282374d6f91d3b958R722-R729) to work with two exclusion field values (`editor__user__groups__name` and `partner__authorization_method`), so this is my attempt at consistency.

Reply:
[Not so consistent I guess](https://github.com/WikipediaLibrary/TWLight/pull/401/files/22bd4f551fbacd0d8722cef8c6d2b0804f474bec#diff-dba635dfa3cab82282374d6f91d3b958R149-R153)

Reply:
it's all Q objects now